pkg/utils/knaming/meta.go

```diff
@@ -22,7 +22,9 @@ const (
 
 	KnTypeLabel = "knative.dev/type"
 
-	FunctionKnType = "function"
+	FunctionKnType  = "function"
+	BaseImageKnType = "baseimage"
```

Comment:
AFAIK, only "function" or "app" are associated with `knative.dev/type` label. We can probably skip this for base image/ image & others.